libs/langchain/langchain/storage/file_system.py

```diff
@@ -36,14 +36,25 @@ class LocalFileStore(ByteStore):
 
     """
 
-    def __init__(self, root_path: Union[str, Path]) -> None:
+    def __init__(
+            self,
+            root_path: Union[str, Path],
```

Comment:
All named arguments with defaults should only appear after `*` to avoid reduce chance of accidental breaking API changes on future refactors

```suggestion
            root_path: Union[str, Path],
            *,
```